test/integration/apiserver/flowcontrol/concurrency_util_test.go

```diff
@@ -138,25 +141,21 @@ type plMetricAvg struct {
 	seatUtil     float64 // average seat utilization
 }
 
-func intervalMetricAvg(snapshots map[string]metricSnapshot, t0 string, t1 string, plLabel string) plMetricAvg {
-	plmT0 := snapshots[t0][plLabel]
-	plmT1 := snapshots[t1][plLabel]
+func intervalMetricAvg(snapshot0 metricSnapshot, snapshot1 metricSnapshot, plLabel string) plMetricAvg {
```

Comment:
s/snapshot0 metricSnapshot, snapshot1 metricSnapshot/snapshot0, snapshot1 metricSnapshot/

Reply:
Changed to suggested in the new version